README.md

```diff
@@ -63,7 +64,7 @@ Cartesian product of the other function parameters--`format`, `compression`, and
 Another example:
 
 ```r
-run_benchmark(write_file, source = "nyctaxi_2010-01", writer = "feather", 
+arrowbench::run_benchmark(write_file, source = "nyctaxi_2010-01", writer = "feather", 
```

Comment:
I don't think this is right. `write_file` is in the `arrowbench` namespace too. IMO we should just say that you need to `library(arrowbench)` first.